src/mongocxx/test/database.cpp

```diff
@@ -33,6 +34,18 @@ using bsoncxx::builder::stream::document;
 using bsoncxx::builder::stream::finalize;
 using bsoncxx::builder::stream::open_document;
 
+namespace {
+
+int get_max_wire_version(const client& mongo_client) {
```

Comment:
Agree, though the test infrastructure is in a somewhat messy state right now, and consequently I don't know where the actual best home for this is yet.  Mind I defer this suggestion to the next time we need to do a wire version check, or when we refactor some of the test infrastructure?